packages/react-nexus/package.json

```diff
@@ -34,15 +34,15 @@
     "url": "https://github.com/BlueBrain/nexus-js/issues"
   },
   "dependencies": {
-    "@bbp/nexus-link": "^1.3.11",
+    "@bbp/nexus-link": "^1.3.12",
```

Comment:
We will have to bump the version of the nexus-link package. The build is failing right now.